cobc/ChangeLog

```diff
@@ -1,4 +1,13 @@
 
+2023-09-04  Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com> and Emilien Lemaire <emilien.lemaire@ocamlpro.com>
```

Comment:
Please wrap that to a second line (also in the libcob one); and as already pointed out: please wither move the entry down to match the date "in order" or update the date.